### ISSUE TITLE
http://go-tracer.appspot.com/ is down

### ISSUE THREAD

**Issue by bennlich** [sjoerdvisscher/GoTracer#1]
I guess it's probably been down for a while? Just came across this repo :-D Sounds super cool.

**Comment by remcoder** [comment]
Yeah it's been down ever since Google's infra went through some update.
We're not doing anything with this at the time so I can't tell you if we'll
ever attempt to bring it up again. Feel free to to fork it and run it on
your own machine or hosting though :-)
